### PR TITLE
Add Fuel Canisters to Contract Composition

### PR DESCRIPTION
**Pull request by TyroneMF**
- Main contract template has 4 fuel canisters
- ~admin\init.sqf has Class EH for carrying.~

### CODE REVIEW

File: template/admin/init.sqf [hunk 1]
```diff
@@ -15,3 +15,15 @@
...
+[
+    "Land_CanisterFuel_F",
+    "init",
+    {
+        [(_this select 0), true, [0, 3, 1], 10] call ACEFUNC(dragging,setCarryable);
+    },
+    true,
+    [],
+    true
+] call CBA_fnc_addClassEventHandler;
```
Comment (jonpas) [inline template/admin/init.sqf:29]:
```suggestion
["Land_CanisterFuel_F", "init", {
    [_this select 0, true, [0, 3, 1], 10] call ACEFUNC(dragging,setCarryable);
}, true, [], true] call CBA_fnc_addClassEventHandler;
```

**Comment by JoramD0** [comment]
What is wrong with the "Take fuel nozzle" interaction?

**Comment by TyroneMF** [comment]
> What is wrong with the "Take fuel nozzle" interaction?

What do you mean?

**Comment by JoramD0** [comment]
Just as you would take a fuel nozzle from a fuel truck you can "Take fuel nozzle" on the fuel canister, but instead of taking just the nozzle it picks up the entirety of the fuel canister.

**Comment by TyroneMF** [comment]
It does work to refuel, you attach the can to the vehicle. Looks a bit weird but still fully functional.

**Pull request merged**